.github/workflows/ocsp-crl-ldap-test.yml

```diff
@@ -330,34 +330,27 @@ jobs:
               -p 8080 \
               -t /ocsp/ee/ocsp \
               -c ca_signing \
-              --serial $CERT_ID \
-              > >(tee stdout) 2> >(tee stderr >&2) || true
+              --serial $CERT_ID | tee output
 
-          # the responder should fail
-          sed -n "s/^SEVERE:\s*\(\S*\)/\1/p" stderr > actual
-          echo "InvalidBERException: Incorrect tag: expected [UNIVERSAL 16], found [UNIVERSAL 28]" > expected
+          # the responder does not provide valid answer
```

Comment:
With this PR the responder will now return "unknown" (which is a valid response) instead of failing, so maybe the comment should say something like this:
```
# the responder should return "unknown"
```